includes/admin/class-edd-notices.php

```diff
@@ -610,6 +580,48 @@ private function add_user_action_notices( $notice = '' ) {
 						'message' => __( 'The order has been updated successfully.', 'easy-digital-downloads' )
 					) );
 					break;
+				case 'order_added' :
+					$this->add_notice( array(
+						'id'      => 'edd-order-added',
+						'message' => __( 'Order successfully created.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'order_trashed' :
+					$this->add_notice( array(
+						'id'      => 'edd-order-trashed',
+						'message' => __( 'The order has been moved to the trash.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'order_restored' :
+					$this->add_notice( array(
+						'id'      => 'edd-order-restored',
+						'message' => __( 'The order has been restored.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'payment_deleted' :
+					$this->add_notice( array(
+						'id'      => 'edd-payment-deleted',
+						'message' => __( 'The order has been deleted.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'email_sent' :
+					$this->add_notice( array(
+						'id'      => 'edd-payment-sent',
+						'message' => __( 'The purchase receipt has been resent.', 'easy-digital-downloads' )
+					) );
+					break;
+				case 'refreshed-reports' :
```

Comment:
This `refreshed-reports` wasn't removed from the `class-edd-notices` file; should it be removed from here (I think this is a duplicate)?

Reply:
Yes that was an oversight and should be removed here. Updated.